### ISSUE TITLE
visSelectNodes mishandles id character vectors of length one

### ISSUE THREAD

**Issue by theRanalyst** [datastorm-open/visNetwork#107]
I'm trying to implement the use of a `selectizeInput` widget for selecting nodes on the network, instead of the `nodeIdSelect` and `selectBy` options and it seems to be causing quite a bit of grief. In the MRE below, you'll see that when you try to use the input from `selectizeInput`, no selection is made and the developer console states that a node with ID [insert first character of string] cannot be found.

MRE:

```{r}
library(shiny)
library(visNetwork)

ui = pageWithSidebar(
  headerPanel('visNetwork visSelectNodes'),
  sidebarPanel(
    selectizeInput(inputId="select_node", label=NULL, multiple=FALSE, 
                   choices=c("20342894", "23492301", "23423442"), selected=NULL, 
                   options=list(placeholder="Search Nodes...",
                                maxOptions=1e5,
                                closeAfterSelect=TRUE))
  ),
  mainPanel(
    visNetworkOutput("graph")
  )
)



server = function(input, output, session) {
  output$graph = renderVisNetwork({
    nodes = data.frame(id=c("20342894", "23492301", "23423442"),
                       color=c("red", "blue", "green"),
                       title=c("20342894", "23492301", "23423442"),
                       label=c("20342894", "23492301", "23423442"),
                       stringsAsFactors=F)
    edges = data.frame(from=c("20342894", "20342894", "23492301"),
                       to = c("23492301", "23423442", "23423442"))
    
    visNetwork(nodes, edges, width="100%") %>%
      visNodes(size=15, borderWidth=0, physics=FALSE) %>%
      visEdges(smooth=FALSE, physics=FALSE, color=list(color="lightgray", highlight="red", hover="red")) %>%
      visOptions(highlightNearest=list(enabled=TRUE, hover=FALSE, degree=1),
                 nodesIdSelection=list(enabled=TRUE, values=unique(nodes$id))) %>%
      visPhysics(enabled=FALSE, stabilization=FALSE) %>%
      visLegend(enabled=FALSE) %>%
      visIgraphLayout(layout="layout.fruchterman.reingold", physics=FALSE, randomSeed=902563)
  })
  
  observe({
    
    selection = input$select_node
    
    visNetworkProxy("graph") %>%
      visUnselectAll() %>%
      visSelectNodes(id=selection, click=FALSE)
  })
}

shinyApp(ui=ui, server=server)
```

Output:

![capture](https://cloud.githubusercontent.com/assets/4400394/19476802/2da7b5ee-9509-11e6-8b80-4ce2446b0805.PNG)

My current work around is to just repeat the value into a non-unique vector of character values, but I'm not a fan of that solution.

**Comment by bthieurmel** [comment]
Fix on new CRAN version 1.0.2

**Comment by theRanalyst** [comment]
So it is. I've been banging my head trying to get things working and didn't even think about updating packages. Thank you for the quick response.
